### DISCUSSION TITLE
I can not see images in README file after changing project visibility from private to public

### DISCUSSION THREAD

**Discussion by falahatiali** [community/community#61586] [Repositories] [answered]
### Select Topic Area

Question

### Body

I had an private project in my repo and changed the visibility from private to public. After that I can not see any images that I had in README. 
I get the following error: pay attention that these images were visible in private repo.
```
AccessDeniedAccess DeniedBWN7RK5KEMK4Q4EA+N+OZ+c6BSEchzp4Fl4gw8lNvR0EJ4KXxm0ofI6Q3iYy/dipySA3MkME//lTnGBicq/aIDdUfyg=
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>BWN7RK5KEMK4Q4EA</RequestId>
<HostId>+N+OZ+c6BSEchzp4Fl4gw8lNvR0EJ4KXxm0ofI6Q3iYy/dipySA3MkME//lTnGBicq/aIDdUfyg=</HostId>
...
</Error>
```

**Comment by moonfury-ops** [comment]
Thank you

**Comment by dimasinr** [comment]
Check Repository Settings: Go to the repository settings on GitHub and make sure that the "Features" section includes the "GitHub Pages" option. If it's enabled, GitHub Pages should allow the images in the README to be displayed.

**Comment by HareshPrajapati** [comment]
If you're experiencing issues with images not being displayed in your README file on GitHub after changing your project's visibility from private to public, here are some steps you can take to troubleshoot the problem:

1. **Cache Refresh:** Sometimes, web browsers store cached versions of web pages. Try refreshing the page or clearing your browser's cache and then reloading the README file.

2. **Check Image Paths:** Ensure that the paths to your images in the README file are correct. Relative paths are commonly used, but make sure they still accurately point to the image files.

3. **Image File Location:** Confirm that the image files are still in the same location within your repository and that they have been included in the public portion of your repository.

4. **GitHub Pages:** If you're using GitHub Pages to host your README, ensure that the image files are available in the GitHub Pages branch (often named `gh-pages` or `docs`).

5. **Case Sensitivity:** GitHub is case-sensitive, so make sure the image file names and their references in the README file match the case exactly.

6. **Markdown Syntax:** Verify that you're using the correct Markdown syntax for including images. Typically, it's `![Alt Text](image_url)`.

7. **Image Hosting:** If the images are hosted externally, make sure the external hosting service allows hotlinking and the images are still accessible.

8. **Public Permissions:** Double-check the permissions of the image files within your repository. They should have public access if you want them to be visible in a public repository.

9. **Third-Party Integrations:** If you're using third-party integrations or tools to manage images, ensure that they are compatible with public repositories.

10. **GitHub Support:** If none of the above steps resolves the issue, consider reaching out to GitHub support or consulting their official documentation for more assistance.

It's worth noting that GitHub's interface and features might have evolved since my last update in September 2021, so make sure to consult the most current information from GitHub's official resources.

**Reply by ghostinhershell** [reply]
@HareshPrajapati While we encourage non-AI answers to questions in community, we also love experimenting and using new technologies. We ask if you do use ChatGPT or similar programs, please disclose what software and model version number you used to generate the response you used in your reply in order to promote transparency in the community.

ChatGPT answers in a tone that sounds like a tech support professional and while we support trying out this software, it is important to verify that they are solutions before posting otherwise it may mislead users into thinking that they are receiving official responses when they are not.

Per ChatGPT's website, the program does have limitations like:
- "May occasionally generate incorrect information"
- "May occasionally produce harmful instructions"
- "Limited knowledge of world and events after 2021"

**Comment by ghufran2508** [comment]
Hi,
I've experienced a similar situation before.
I solved this by re-uploading the images by dragging and dropping them into your README.md and updating only the image links where necessary. This time github mark this asset as public and the access denied problem will no longer be there.

**Comment by b8zeek** [comment, marked as answer]
Hey @falahatiali,

You can definitely solve this issue by uploading the images directly to the repository and just point to them in the README.md file. Take a look at my readme repository for the reference:

https://github.com/bejzik8/bejzik8/tree/master/src

Good luck. 🤞🏼 

🍀

**Reply by SshibamDebnathh** [reply]
thankyou I was also getting issues but your answer really helped
